Makefile.toml

```diff
@@ -212,7 +214,13 @@ args = ["component", "add", "rust-src"]
 category = "Test"
 description = "Install cargo-component extension"
 command = "cargo"
-args = ["install", "cargo-component@0.7.0"]
+args = ["install", "cargo-component@0.14.0"]
+
+[tasks.install-cargo-tarpaulin]
```

Comment:
You don't actually need this task, if you have a task that uses `cargo` as the command, `cargo-make` will check if the task name passed to `cargo` exists, and if not, it will attempt to `cargo install` it.

There's also `cargo-make` configuration for modifying how `cargo install` is invoked, like if the crate name is different, or additional arguments are needed. Not needed here, but just FYI.

The only reason to explicitly install things is when those things aren't normal `cargo` plugins, e.g. `mdbook`